package.json

```diff
@@ -10,7 +10,7 @@
     "console": "sst console",
     "lint": "tsc && eslint '**/*.js'",
     "clean": "rm -rf docs node_modules pnpm-lock.yaml packages/*/{pnpm-lock.yaml,coverage,.nyc_output,dist,node_modules,.next}",
```

Comment:
unrelated note: the clean target is wild. we should drop it. deleting the docs folder and a bunch of pnpm and next stuff seems odd. I defo dont want to delete the docs folder now we have manually created docs